features/kinderen.feature

```diff
@@ -77,84 +86,86 @@ Functionaliteit: Kinderen van een persoon raadplegen
     - Wanneer door de gebruikte fields parameter in het request het kind in de response geen enkel gegeven heeft met een waarde, dan wordt het kind geleverd zonder gegevens (dus als leeg object)
 
     @gba
-    Abstract Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
-      Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
-      | 9         |                             |                   |                     | <geslachtsnaam>       | <geboortedatum>       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+    Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
+      Gegeven de persoon met burgerservicenummer '000003456' heeft een 'kind' met de volgende gegevens
+      | geslachtsnaam (02.40) | 
+      | .                     |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 000003456                       |
       | fields              | kinderen.naam                   |
-      Dan heeft de persoon met burgerservicenummer '999996150' een kind met de volgende 'naam' gegevens
-      | naam          | waarde          |
-      | geslachtsnaam | <geslachtsnaam> |
-
-      Voorbeelden:
-      | geslachtsnaam | geboortedatum |
-      | .             |               |
-      | .             | 00000000      |
+      Dan heeft de response een persoon met een 'kind' met de volgende gegevens
```

Comment:
Dit is de response (zonder de type) die de GBA API moet geven. De proxy zou volgens mij de volgende response moeten geven:
```
{
  "kinderen": [
     {
        "naam": {}
     }
  ]
}
```
We hebben nu de stap `heeft de response een persoon met een '\<naam collectie enkelvoud>' zonder gegevens`. Conform deze stap definitie zal ik de volgende stap definitie `heeft de response een persoon met een '\<naam collectie enkelvoud>' zonder '<\naam gegevensgroep>' gegevens` toevoegen om bovenstaande json te kunnen valideren. Voor deze scenario ziet de Dan stap er als volgt uit: `Dan heeft de response een persoon met een 'kind' zonder 'naam' gegevens`

Reply:
Top, wacht ik tot je dat toegevoegd hebt.

Reply:
> Top, wacht ik tot je dat toegevoegd hebt.

De stap is nu toegevoegd

Reply:
Ik heb zojuist de laatste versie van de chore/test-e2e-automation branch opgehaald maar ik krijg op het scenario 'kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields' nog het volgende resultaat terug van de proxy: 

`[ ]`

Wellicht heeft ook dit te maken met waar je voor je vakantie mee bezig was.